change/@fluentui-react-button-2cbb912a-03db-43b4-b4b5-9689954351b0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 73 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26522&commitId=fff7e239d6e29ebaab7f5f27f9c797c122b118d9&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1796 screenshots unchanged, 0 screenshots with different dimensions and 73 screenshots with visible difference.


  <details><summary><b>unknown</b> 73 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Converged.Circular and icon only - RTL.default.chromium.png|69| Changed|
|Button Converged.Circular and icon only - RTL.hover.chromium.png|50| Changed|
|Button Converged.Circular and icon only - RTL.pressed.chromium.png|51| Changed|
|Button Converged.Circular and icon only.default.chromium.png|69| Changed|
|Button Converged.Circular and icon only.hover.chromium.png|50| Changed|
|Button Converged.Circular and icon only.pressed.chromium.png|51| Changed|
|Button Converged.Icon only.default.chromium.png|68| Changed|
|Button Converged.Icon only.hover.chromium.png|48| Changed|
|Button Converged.Icon only.pressed.chromium.png|50| Changed|
|Button Converged.Size large - with long text wrapping.chromium.png|1614| Changed|
|Button Converged.Size large.default.chromium.png|263| Changed|
|Button Converged.Size large.hover.chromium.png|245| Changed|
|Button Converged.Size large.pressed.chromium.png|256| Changed|
|Button Converged.Size medium - with long text wrapping.chromium.png|861| Changed|
|Button Converged.Size small - with long text wrapping.chromium.png|875| Changed|
|Button Converged.Size small.default.chromium.png|248| Changed|
|Button Converged.Size small.hover.chromium.png|240| Changed|
|Button Converged.Size small.pressed.chromium.png|266| Changed|
|Card Converged.CardHeader.normal.chromium.png|4026| Changed|
|CompoundButton Converged.Size small - with long text wrapping.chromium.png|2| Changed|
|CompoundButton Converged.Size small.default.chromium.png|2| Changed|
|CompoundButton Converged.Size small.hover.chromium.png|2| Changed|
|CompoundButton Converged.Size small.pressed.chromium.png|2| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|3215| Changed|
|Dialog.title custom action - High Contrast.chromium.png|5218| Changed|
|Dialog.title custom action - RTL.chromium.png|3478| Changed|
|Dialog.title custom action.chromium.png|3485| Changed|
|MenuButton Converged.Circular and icon only.default.chromium.png|69| Changed|
|MenuButton Converged.Circular and icon only.hover.chromium.png|50| Changed|
|MenuButton Converged.Circular and icon only.pressed.chromium.png|51| Changed|
|MenuButton Converged.Icon only.default.chromium.png|68| Changed|
|MenuButton Converged.Icon only.hover.chromium.png|48| Changed|
|MenuButton Converged.Icon only.pressed.chromium.png|50| Changed|
|MenuButton Converged.Size large - with long text wrapping.chromium.png|1635| Changed|
|MenuButton Converged.Size large.default.chromium.png|284| Changed|
|MenuButton Converged.Size large.hover.chromium.png|266| Changed|
|MenuButton Converged.Size large.pressed.chromium.png|280| Changed|
|MenuButton Converged.Size medium - with long text wrapping.chromium.png|870| Changed|
|MenuButton Converged.Size small - with long text wrapping.chromium.png|888| Changed|
|MenuButton Converged.Size small.default.chromium.png|265| Changed|
|MenuButton Converged.Size small.hover.chromium.png|258| Changed|
|MenuButton Converged.Size small.pressed.chromium.png|289| Changed|
|ToggleButton Converged.Circular and icon only.default.chromium.png|69| Changed|
|ToggleButton Converged.Circular and icon only.hover.chromium.png|50| Changed|
|ToggleButton Converged.Circular and icon only.pressed.chromium.png|51| Changed|
|ToggleButton Converged.Icon only.default.chromium.png|68| Changed|
|ToggleButton Converged.Icon only.hover.chromium.png|48| Changed|
|ToggleButton Converged.Icon only.pressed.chromium.png|50| Changed|
|ToggleButton Converged.Size large - with long text wrapping.chromium.png|1614| Changed|
|ToggleButton Converged.Size large.default.chromium.png|263| Changed|
|ToggleButton Converged.Size large.hover.chromium.png|245| Changed|
|ToggleButton Converged.Size large.pressed.chromium.png|256| Changed|
|ToggleButton Converged.Size medium - with long text wrapping.chromium.png|861| Changed|
|ToggleButton Converged.Size small - with long text wrapping.chromium.png|875| Changed|
|ToggleButton Converged.Size small.default.chromium.png|248| Changed|
|ToggleButton Converged.Size small.hover.chromium.png|240| Changed|
|ToggleButton Converged.Size small.pressed.chromium.png|266| Changed|
|Toolbar Converged.Default.Button Pressed.chromium.png|68| Changed|
|Toolbar Converged.Default.Toggle On.chromium.png|64| Changed|
|Toolbar Converged.Default.default.chromium.png|62| Changed|
|Toolbar Converged.Far Group.chromium.png|186| Changed|
|Toolbar Converged.Large.Button Pressed.chromium.png|72| Changed|
|Toolbar Converged.Large.Toggle On.chromium.png|71| Changed|
|Toolbar Converged.Large.default.chromium.png|68| Changed|
|Toolbar Converged.Small.Button Pressed.chromium.png|139| Changed|
|Toolbar Converged.Small.Toggle On.chromium.png|147| Changed|
|Toolbar Converged.Small.default.chromium.png|147| Changed|
|Toolbar Converged.Vertical Button.Button Pressed.chromium.png|53| Changed|
|Toolbar Converged.Vertical Button.Toggle On.chromium.png|43| Changed|
|Toolbar Converged.Vertical Button.default.chromium.png|43| Changed|
|Toolbar Converged.Vertical.Button Pressed.chromium.png|5052| Changed|
|Toolbar Converged.Vertical.Toggle On.chromium.png|5055| Changed|
|Toolbar Converged.Vertical.default.chromium.png|5062| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
All changed screenshots are intended.